doc/changes/changes_1.0.1.md

```diff
@@ -0,0 +1,29 @@
+# hamcrest-multiline-text-matcher 1.0.1, released 2020-12-19
+
+Codename: Update Frenzy
+
+## Summary 
+
+In version 1.0.1 we ported all tests from JUnit 4 to JUnit 5. We also updated all other dependencies.
```

Comment:
Breaking change: upgrade from java 8 to 11

Reply:
Added. Renamed to 2.0.0.